api.swagger.yaml

```diff
@@ -1682,6 +1682,161 @@ paths:
         '422':
           description: JSON schema validation failure
       description: Report a post as the current user.
+  '/lists':
+    post:
+      tags:
+        - 'lists'
+      description: 'Provision a new list.'
+      responses:
+        '201':
```

Comment:
should we also return a response body with the uuid?

Reply:
put in same way as https://io1dev.atlassian.net/browse/ITB-1136